nyaa/templates/layout.html

```diff
@@ -44,8 +44,174 @@
 		<![endif]-->
 	</head>
 	<body>
+		{% set nyaa_cats = [

```

Comment:
How about this?

![2017-05-23_023319](https://cloud.githubusercontent.com/assets/9733856/26323119/394dcea4-3f60-11e7-8b68-5e0e2d645cb1.png)